### DISCUSSION TITLE
E3V2-ManualMesh-5x5-v4.2.7-v1.3.4 Config

### DISCUSSION THREAD

**Discussion by CoolChriz1984** [Jyers/Marlin#1122] [Q&A] [answered]
Hello. I need the firmware but with a max bedtem of 140° and hotendtemp of 280°. I tried to compile it myself but it went wrong. The printer hangs up. Black screen. I then reinstalled the normal release and it works again. But I need the high temperatures. Can someone help me?

**Comment by Jyers** [comment, marked as answer]
When compiling your own did you start with a template config found in the configuration files folder?

**Reply by CoolChriz1984** [reply]
Hi. I start with a template

**Reply by Jyers** [reply]
Perhaps try reformatting your sd card
